pageserver/src/tenant/ephemeral_file.rs

```diff
@@ -256,12 +199,12 @@ impl BlobWriter for EphemeralFile {
 
 impl Drop for EphemeralFile {
     fn drop(&mut self) {
-        // drop all pages from page cache
-        let cache = page_cache::get();
-        cache.drop_buffers_for_ephemeral(self.file_id);
-
```

Comment:
This would still make sense to do, no?

Reply:
I have an issue about this, #3769, but when writing it, I did not realize how ephemeral files worked.

Reply:
Correct, we still need this, but, need to use `drop_buffers_for_immutable`.

Reply:
Fixed in 784ff0aa1e2fe2c9cf45dd0037fe44088ce41e32

@koivunej the issue #3769 that you mentioned is still valid after this change, and I don't intend to fix it as part of this change.